### PR TITLE
Workin on Cassandra 3.7 support

### PR DESCRIPTION
**Pull request by Jan-M**

### CODE REVIEW

**Comment by Jan-M** [comment]
@smourapina thx for your input from 3.0.8 I basically went on from there

File: Dockerfiles/Scylla-1.3/start-scylla [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+fi
+
+sed -i "s/seeds: \"127.0.0.1\"/seeds: \"$SEEDS\"/g" /etc/scylla/scylla.yaml
+sed -i "s/listen_address: localhost/listen_address: $LISTEN_ADDRESS/g" /etc/scylla/scylla.yaml
+
+if [ x"$SCYLLA_BROADCAST_ADDRESS" != "x" ];then
+	sed -i "s/.*broadcast_address:.*/broadcast_address: $SCYLLA_BROADCAST_ADDRESS/g" /etc/scylla/scylla.yaml
+fi
+
+/usr/bin/scylla --log-to-syslog 0 --log-to-stdout 1 $DEV_MODE $SEASTAR_IO $CPU_SET --default-log-level info --options-file /etc/scylla/scylla.yaml --listen-address $LISTEN_ADDRESS --rpc-address $LISTEN_ADDRESS --network-stack posix &
```
Comment (a1exsh) [inline Dockerfiles/Scylla-1.3/start-scylla]: Should it be the same `$CPUSET` (no underscore?)
Why set LISTEN_ADDRESS both in config and on command line?

File: Dockerfiles/Scylla-1.3/start-scylla [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+source /etc/default/scylla-jmx
+export SCYLLA_HOME SCYLLA_CONF
+exec /usr/lib/scylla/jmx/scylla-jmx -l /usr/lib/scylla/jmx &
+
+# not perfect, also waits for scylla-jmx only, with scylla not running
+wait
```
Comment (a1exsh) [inline Dockerfiles/Scylla-1.3/start-scylla]: `wait %1`, but still ugly

File: Dockerfiles/Scylla-1.3/start-scylla [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+if [ x"$SCYLLA_BROADCAST_ADDRESS" != "x" ];then
+	sed -i "s/.*broadcast_address:.*/broadcast_address: $SCYLLA_BROADCAST_ADDRESS/g" /etc/scylla/scylla.yaml
+fi
+
+/usr/bin/scylla --log-to-syslog 0 --log-to-stdout 1 $DEV_MODE $SEASTAR_IO $CPU_SET --default-log-level info --options-file /etc/scylla/scylla.yaml --listen-address $LISTEN_ADDRESS --rpc-address $LISTEN_ADDRESS --network-stack posix &
+
+source /etc/default/scylla-jmx
+export SCYLLA_HOME SCYLLA_CONF
+exec /usr/lib/scylla/jmx/scylla-jmx -l /usr/lib/scylla/jmx &
```
Comment (a1exsh) [inline Dockerfiles/Scylla-1.3/start-scylla]: `exec` here seems like not very useful

**Comment by a1exsh** [comment]
I would still like to use branches instead of having all different variants in the master.

If you keep everything in master, every feature merged to master will trigger the build of 3 images: cassandra-2.x, cassandra-3.x and now also scylladb-1.x.  Even if the change was only targeted at only one of them.

It is not very likely IMO that there will be a lot of changes that affect a number of versions at the same time.  But by updating a specific branch you could trigger a build of exactly that version.  Moreover, we can exclude the admin script from the versioned branches and only keep it in master: then any changes to `create_cluster.py` will not trigger unnecessary builds of any docker images.

**Pull request merged**